wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -17,54 +17,8 @@
     {% sidebar_collapsed as sidebar_collapsed %}
     <body id="wagtail" class="{% block bodyclass %}{% endblock %} {% if sidebar_collapsed %}sidebar-collapsed{% endif %} {% if messages %}has-messages{% endif %}">
         <div data-sprite></div>
-        <script>
-            function loadIconSprite() {
-                var spriteURL = '{% url "wagtailadmin_sprite" %}';
-                var revisionKey = 'wagtail:spriteRevision';
-                var dataKey = 'wagtail:spriteData';
-                var isLocalStorage = 'localStorage' in window && typeof window.localStorage !== 'undefined';
 
-                var insertIt = function (data) {
-                    var spriteContainer = document.body.querySelector('[data-sprite]');
-                    spriteContainer.innerHTML = data;
-                }
-
-                var insert = function (data) {
-                    if (document.body) {
-                        insertIt(data)
-                    } else {
-                        document.addEventListener('DOMContentLoaded', insertIt.bind(null, data));
-                    }
-                }
-
-                if (isLocalStorage && localStorage.getItem(revisionKey) === spriteURL) {
-                    var data = localStorage.getItem(dataKey);
-                    if (data) {
-                        insert(data);
-                        return true;
-                    }
-                }
-
-                try {
-                    var request = new XMLHttpRequest();
```

Comment:
I recommend we migrate this to `fetch` - it does not look like we need to use `XMLHttpRequest` and it probably was written that way because the code was not transpiled.

https://medium.com/beginners-guide-to-mobile-web-development/the-fetch-api-2c962591f5c